index.html

```diff
@@ -37,7 +37,7 @@
     readonly        attribute DOMString sinkId;
     Promise&lt;void&gt; setSinkId (DOMString sinkId);
 };</pre><section><h2>Attributes</h2><dl data-link-for="HTMLMediaElement" data-dfn-for="HTMLMediaElement" class="attributes"><dt><dfn><code>sinkId</code></dfn> of type <span class="idlAttrType"><a>DOMString</a></span>, readonly       </dt><dd>
-        The unique ID of the audio device through which output is being delivered, or empty string if it is using the user agent default. This ID should be one of the <a href="https://w3c.github.io/mediacapture-main/#widl-MediaDeviceInfo-deviceId"><code>MediaDeviceInfo.deviceId</code></a> values returned from <a href="https://w3c.github.io/mediacapture-main/#widl-MediaDevices-enumerateDevices-void-MediaDeviceInfoCallback-resultCallback"><code>MediaDevices.enumerateDevices()</code></a> [[!GETUSERMEDIA]], or one of the predefined IDs listed below.
+        The unique ID of the audio device through which output is being delivered, or empty string if it is using the user agent default. This ID should be equal to the <a href="https://w3c.github.io/mediacapture-main/#dom-mediadeviceinfo-deviceid"><code>deviceId</code></a> attribute of one of the <a href="https://w3c.github.io/mediacapture-main/#idl-def-mediadeviceinfo"><code>MediaDeviceInfo</code></a> values returned from <a href="https://w3c.github.io/mediacapture-main/#dom-mediadevices-enumeratedevices"><code>MediaDevices.enumerateDevices()</code></a> [[!GETUSERMEDIA]], or one of the predefined IDs listed below.
```

Comment:
Nit: "or empty string" -> "or the empty string". You might want to turn on auto line breaks; this line is very long.